src/jet_client.rs

```diff
@@ -14,21 +18,21 @@ use jet_proto::{
 };
 use slog_scope::{debug, error};
 use tokio::{
-    io::{AsyncRead, AsyncWrite},
-    runtime::TaskExecutor,
+    io::{AsyncRead, AsyncWrite, ReadBuf},
+    sync::Mutex,
 };
 use uuid::Uuid;
 
 use crate::{
     config::Config,
-    http::controllers::jet::create_remove_association_future,
+    http::controllers::jet::remove_association,
     jet::{
         association::Association,
         candidate::{Candidate, CandidateState},
         TransportType,
     },
     transport::JetTransport,
-    utils::association::{RemoveAssociation, ACCEPT_REQUEST_TIMEOUT_SEC},
+    utils::association::{RemoveAssociation, ACCEPT_REQUEST_TIMEOUT},
     Proxy,
 };
 
```

Comment:
JetAssociationsMap above was changed to tokio Mutex, which should play more nicely with async code